features/kinderen.feature

```diff
@@ -83,9 +83,20 @@ Functionaliteit: Kinderen van een persoon raadplegen
       Voorbeelden:
       | geslachtsnaam | geboortedatum |
       | .             |               |
-      |               | 00000000      |
       | .             | 00000000      |
 
+      @gba
+      Abstract Scenario: kind volledig onbekend, veld met onbekend waarde wordt niet gevraagd met fields
```

Comment:
dit is geen abstract scenario, want heeft geen voorbeelden

Reply:
Aangepast